<?xml version="1.0"?>
<launch>
    <group ns="ur5_controller_master">
        <rosparam file="$(find ur5_controller)/config/ur5_controller_global_config.yaml" command="load" subst_value="true"/>
        <node pkg="ur5_controller" type="ur5_controller_master_node" name="ur5_controller_master_node" output="screen"/>
    </group>
</launch>